product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/12_pem_manage_probes/03_pem_probe_config/01_pem_probes.mdx

```diff
@@ -9,11 +9,17 @@ legacyRedirectsGenerated:
 | ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------- |
 | Background Writer Statistics             | This probe monitors information about the background writer. The information includes:<br />-   The number of timed checkpoints     <br /> -   The number of requested checkpoints     <br /> -   The number of buffers written (by checkpoint)     <br /> -   The number of buffers written (by background writer)     <br /> -   The number of background writer cycles     <br /> -   The number of background buffers written     <br /> -   The number of buffers allocated     <br />                                                                                                                                                                                                                                    | Server                              |
+| Barman Configuration                     | This probe returns information about Barman tool global configuration.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Agent                    |
```

Comment:
add "the" after "about" in these descriptions

Reply:
done in a separate commit